### DISCUSSION TITLE
Payment with new PayPal  Method

### DISCUSSION THREAD

**Discussion by GiovanniMorelli** [PrestaShop/PrestaShop#33270] [Q&A] [answered]
Hosting => Aruba
PHP version  => 8.1
Prestashop version => 8.1.0

Hi, I'm trying to pay with the PayPal method, but when I click on the "send" button, the payment on  PayPal goes in success (I found the transaction on PayPal) but the response on Prestashop goes in error.

I have inspected the network on Google, and I found the method called "validate" has an Error in response. 
Thanks for your support.

<img width="800" alt="image" src="https://github.com/PrestaShop/PrestaShop/assets/17708511/99187adf-502b-40ce-9adc-4e77e3afaf9b">

**Comment by MatShir** [comment, marked as answer]
Hi @GiovanniMorelli,

We use GitHub issues only to discuss bugs and new features about the PrestaShop project, not the products from PrestaShop company or Partners Module. For Checkout, there's an email address where you can explain your problem : [support-checkout-download@prestashop.com](mailto:support-checkout-download@prestashop.com). They'll get back to you faster than the community :)

**Comment by angusin** [comment]
@GiovanniMorelli we are experiencing the same problem. Did you find any solution or workaround? Thanks!

**Reply by GiovanniMorelli** [reply]
Hi @angusin I have rollback to 1.7.8.9 and it works. I think  it's a problem with template Html Prestashop or Php version

**Reply by angusin** [reply]
Thank you so much @GiovanniMorelli !! What a shitty release it has been the PS 8 version.

**Reply by boherm** [reply]
Hi @angusin, I'm really sorry to read that you're are disappointed by PS 8.
But if you want to make things better, feel free to help us by contributing and testing the next major release!
I really hope to see you helping us to improve this great project and make PS more stable and reliable!
